src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/draggableList/sortableItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {DraggableSyntheticListeners} from '@dnd-kit/core';
+import {useSortable} from '@dnd-kit/sortable';
+import {Transform} from '@dnd-kit/utilities';
+
+type Props = {
+  id: string;
+  renderItem(args: {
+    value: React.ReactNode;
+    listeners?: DraggableSyntheticListeners;
+    transform?: Transform | null;
+    transition?: string;
+    style?: React.CSSProperties;
+    forwardRef?: React.Ref<HTMLElement>;
+  }): React.ReactElement | null;
+};
+
+function SortableItem({id, renderItem}: Props) {
```

Comment:
I have to have this Component because it uses hook and a hook has to be at the Top Level